### ISSUE TITLE
Add an article already added notification

### ISSUE THREAD

**Issue by emporeso2** [wallabag/android-app#421]
Hi all,

i'am usiing wallabag android app version 1.12.0(30) with wallabag.it service.
When i add an article twince in the app,
the app don't send me notification to tell me that my article were already added ?

**Comment by di72nn** [comment]
Yes, it does not. The app only shows that the link was added. There's a couple of reasons behind it.

First of all, it is not guaranteed to work with the current implementation: the app uses RSS feeds which may be limited on the server side, so the app might not contain the article to determine that it is already present.

Another reason is that I didn't the app want to perform additional DB queries while the app was in background.

That feature should be considered again after v2 release.
